src/routes/login/LoginView.vue

```diff
@@ -185,6 +185,7 @@ import BaseSnackbar from '@/components/BaseSnackbar.vue';
 import MarkdownContent from '@/components/MarkdownContent.vue';
 import BaseVideo from '@/components/BaseVideo.vue';
 import TheLanguageSelector from '@/components/TheLanguageSelector.vue';
+import { useHead } from '@unhead/vue';
```

Comment:
Please move the import above the internal ones so it goes with the external deps

Reply:
Hey @itsalaidbacklife I moved the import above the internal imports. Here is the [link](https://github.com/cuttle-cards/cuttle/pull/1145/commits/48d0a2b74769d1349504ff790ff284668be7a75b)